eldritch/eldritch.py

```diff
@@ -772,6 +772,7 @@ def handle_ancient(self, ancient):
     if ancient not in self.all_ancients:
       raise InvalidMove(f"Unknown ancient one {ancient}")
     self.ancient_one = self.all_ancients[ancient]
+    self.ancient_one.setup(self)
```

Comment:
Can I ask you to move this from handle_ancient to handle_start? If we want to do something like adding masks, we don't want to have to remove the masks when the player changes their minds about which ancient one to fight.